packages/node/features/fixtures/sessions/scenarios/start-session-notify.js

```diff
@@ -0,0 +1,17 @@
+var bugsnag = require('@bugsnag/node')
+var bugsnagClient = bugsnag({
+  apiKey: process.env.BUGSNAG_API_KEY,
+  endpoints: {
+    notify: process.env.BUGSNAG_NOTIFY_ENDPOINT,
+    sessions: process.env.BUGSNAG_SESSIONS_ENDPOINT
+  },
+  sessionSummaryInterval: 1000
+})
+
+var sessionClient = bugsnagClient.startSession()
+setTimeout(function () {
+  sessionClient.notify(new Error('in a session'))
+}, 1500)
+
+// keep the process open a little bit so that the session has time to get sent
+setTimeout(function () {}, 5000)
```

Comment:
Is there any way of extracting a constant out of 1000/5000 across these fixtures? We may want to tweak this value in future

Reply:
I think it depends on the test and I'm not keen on factoring this out unless it is duplicated at least 3x